WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -584,25 +613,44 @@ - (UITableViewCell *)configureMetaPostMetaCellForIndexPath:(NSIndexPath *)indexP
         cell.accessibilityIdentifier = @"Visibility";
 
     } else {
-        // Password
-        WPTextFieldTableViewCell *textCell = [self getTextFieldCell];
-        textCell.textLabel.text = NSLocalizedString(@"Password", @"Label for the tags field. Should be the same as WP core.");
-        textCell.textField.text = self.apost.password;
-        textCell.textField.attributedPlaceholder = nil;
-        textCell.textField.placeholder = NSLocalizedString(@"Enter a password", @"");
-        textCell.textField.clearButtonMode = UITextFieldViewModeWhileEditing;
-        textCell.textField.secureTextEntry = YES;
-
-        cell = textCell;
-        cell.tag = PostSettingsRowPassword;
-        
-        self.passwordTextField = textCell.textField;
-        self.passwordTextField.accessibilityIdentifier = @"Password Value";
+        cell = [self configurePasswordCell];
     }
 
     return cell;
 }
 
+- (UITableViewCell *)configurePasswordCell
+{
+    // Password
+    WPTextFieldTableViewCell *textCell = [self getTextFieldCell];
+    textCell.textLabel.text = NSLocalizedString(@"Password", @"Label for the tags field. Should be the same as WP core.");
+    textCell.textField.text = self.apost.password;
+    textCell.textField.attributedPlaceholder = nil;
+    textCell.textField.placeholder = NSLocalizedString(@"Enter a password", @"");
+    textCell.textField.clearButtonMode = UITextFieldViewModeWhileEditing;
+    textCell.textField.secureTextEntry = YES;
+    
+    textCell.tag = PostSettingsRowPassword;
+    
+    self.passwordTextField = textCell.textField;
+    self.passwordTextField.accessibilityIdentifier = @"Password Value";
+    
+    [self configureVisibilityButtonForPasswordCell:textCell];
+    
+    return textCell;
+}
+
+- (void)configureVisibilityButtonForPasswordCell:(WPTextFieldTableViewCell *)textCell
+{
+    self.passwordVisibilityButton = [UIButton buttonWithType:UIButtonTypeCustom];
+    self.passwordVisibilityButton.frame = CGRectMake(0, 0, 24, 20);
```

Comment:
Can we pass the image size here instead of using magic numbers? Or will the button size to fit its image anyway?